### PR TITLE
Adding App list page to minimal and base menus

### PR DESCRIPTION
**Pull request by speizerj**
Minimal Menu displays the app list like so:

image

This page uses the `eager_options_handle` slug for two reasons:

1. The specific use case where a site has 2 apps installed, so the Base menu is used, and they uninstall one of the apps. CMSIntegration then redirects the user to `eager_options_handle`, which is normally the main Eager settings page under Base menu, but because they only have 1 app installed, the Minimal menu is displayed. So they will be taken back to the app list page. I think this UX makes sense. If we don't use this slug, the user would get a "permission denied" issue unless we added in the settings page as a menu item, which seems unecessary. 
2. (This is a bit of a bonus) The Eager Logo is set to be used only on menu items that have the `eager_options_handle` slug. Easy to change this, but it's nice that it works out

The base menu is the same as it was before:

image

### CODE REVIEW

**Comment by speizerj** [comment]
Keep in mind this needs this PR to be merged for this to work:

https://github.com/EagerIO/CMSIntegration/pull/35

File: includes/settings.php [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+    add_submenu_page('eager_options_handle', $app['title'], $app['title'] . ' (inactive)', 'activate_plugins',  'eager_app_'.$appId.'_options', 'eager_install_options');
+  }
+
+  if ($installs){
+    foreach ($installs as $install) {
+      add_submenu_page('eager_options_handle', $install['app']['title'], $install['app']['title'], 'activate_plugins',  'eager_app_'.$install['appId'].'_options', 'eager_install_options');
+    }
+  }
+
+  //add back in once it's ready
```
Comment (zackbloom) [inline includes/settings.php]: I believe this comment isn't relevant anymore.

File: includes/settings.php [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+  }
+
+  $ip = gethostbyname($_SERVER['SERVER_NAME']);
+  $proxy = array(
+    'port' => '',
+    'version' => ''
+  );
+
+  if ($ip == '127.0.0.1') {
+    $proxy = eager_get_proxy_info();
```
Comment (zackbloom) [inline includes/settings.php]: All this proxy stuff was killed no? I think you need to merge master -> this branch.

**Comment by speizerj** [comment]
@zackbloom I've tested against all the different versions listed on this page: https://wordpress.org/plugins/drift-by-eager/developers/

Everything appears to be working as expected, both locally and on Tangy. It'd be great if you could give it a spin before we take it live - would be good to have a second set of eyes on it.

**Comment by zackbloom** [comment]
LGTM, :ship:!

**Comment by speizerj** [comment]
Using `cmsv2.0` instead, closing and deleting

**Pull request closed without merging**